### PR TITLE
Latest labeler notebooks

### PR DESCRIPTION
**Pull request by shrutix**
Has both new article data (not 600 article dataset yet) and old wiki data, commented out code can be used interchangeably 
Includes code to extract model into .bin file

### CODE REVIEW

**Comment by shrutix** [comment]
Does not contain tarball or aws sagemaker code (files too large for github)

**Comment by pjuangph** [comment]
@shrutix are you using both pytorch and tensorflow? you have your gpu detection code using both. This is common in all your .pynb files. Please remove the one that you are not using. 

auto_labeler_xlnet
What is the attention_masks? is that needed? 

Your number of labels for BERT appears to be 5 but i see that forloop you have above where you set labels. That's more than 5. Can you explain this? 

Are you training the entire bert model ? 

At the end you reinitialize bert? You are doing this after you went through all the trouble of training it. I am confused. 
```
# Initializing a BERT bert-base-uncased style configuration
configuration = BertConfig()

# Initializing a model from the bert-base-uncased style configuration
model = BertModel(configuration)

model = BertModel.from_pretrained('bert-base-uncased', return_dict=True)
```

So long ago I posted in technical focus https://biomindset.slack.com/archives/CNQJY2GG3/p1613794149000700 What did you think of this approach? would you be able to give it a try? I see the accuracy of the test set is 0.11 is that a percentage or do you have to multiply it to be a percentage? this is low either way. I think we should try a different approach all together.

**Comment by shrutix** [comment]
> @shrutix are you using both pytorch and tensorflow? you have your gpu detection code using both. This is common in all your .pynb files. Please remove the one that you are not using.
> 
> auto_labeler_xlnet
> What is the attention_masks? is that needed?
> 
> Your number of labels for BERT appears to be 5 but i see that forloop you have above where you set labels. That's more than 5. Can you explain this?
> 
> Are you training the entire bert model ?
> 
> At the end you reinitialize bert? You are doing this after you went through all the trouble of training it. I am confused.
> 
> ```
> # Initializing a BERT bert-base-uncased style configuration
> configuration = BertConfig()
> 
> # Initializing a model from the bert-base-uncased style configuration
> model = BertModel(configuration)
> 
> model = BertModel.from_pretrained('bert-base-uncased', return_dict=True)
> ```
> 
> So long ago I posted in technical focus https://biomindset.slack.com/archives/CNQJY2GG3/p1613794149000700 What did you think of this approach? would you be able to give it a try? I see the accuracy of the test set is 0.11 is that a percentage or do you have to multiply it to be a percentage? this is low either way. I think we should try a different approach all together.

reiterating my responses here for reference:
- removing the first cell that checks for GPU using tf
- issues with initializing bert are from old code, now deleted
- -xlnet model notebook deleted

**Comment by pjuangph** [comment]
The code is fine.
